lxd/api_internal.go

```diff
@@ -588,8 +588,8 @@ func internalSQLExec(tx *sql.Tx, query string, result *internalSQLResult) error
 
 // internalImportFromBackup creates instance, storage pool and volume DB records from an instance's backup file.
 // It expects the instance volume to be mounted so that the backup.yaml file is readable.
-func internalImportFromBackup(s *state.State, projectName string, instName string, allowNameOverride bool) error {
-	if instName == "" {
+func internalImportFromBackup(s *state.State, srcBackup *backup.Info, allowNameOverride bool) error {
```

Comment:
Is this required, all other config data in this function is retrieved from  backup.ParseConfigYamlFile below, which should contain the volume's volatile.uuid too? Want to keep the config location consistent for all config if possible.

Reply:
Not required anymore using your suggestion from https://github.com/canonical/lxd/pull/12904#discussion_r1492977454.

Reply:
This is removed from the PR too.